src/Symfony/Component/Form/FormFactory.php

```diff
@@ -73,6 +73,7 @@ public function createBuilder($type = 'form', $data = null, array $options = arr
     public function createNamedBuilder($name, $type = 'form', $data = null, array $options = array())
```

Comment:
I'm not sure about this, `$data` can be anything and `$options` is optional, so this can be a BC break if someone pass an array of data and don't configure any option. Right?